### DISCUSSION TITLE
TypeError issue with example script

### DISCUSSION THREAD

**Discussion by ProThicc** [jcmgray/quimb#206] [Q&A] [answered]
Hello,
when testing the example script here https://quimb.readthedocs.io/en/latest/distributed%20parallelism%20-%20mpi.html, 
I get the error TypeError: Cannot cast array data from dtype('complex128') to dtype('float64') according to the rule 'safe'. I am pretty new to slepc methods actually and did a conda install for both petsc and slepc. If anyone had faced the same issue and was able to bypass it, could you please help me out?

Thanks

**Comment by jcmgray** [comment, marked as answer]
If you are trying to solve a complex Hamiltonian, you will need the version itself of slepc installed to match. If you look on the conda forge page for slepc4py there should be some pointers for installing the (non-default) complex version. I suspect that may be it!

**Reply by ProThicc** [reply]
Thank you! That fixed the issue. I was installing the real build.
